lib/OrganizationsFieldArray/OrganizationPicker.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { TextField } from '@folio/stripes/components';
+import { Pluggable } from '@folio/stripes/core';
+
+export default class OrganizationPicker extends React.Component {
+  static propTypes = {
+    addOrganization: PropTypes.func,
+    index: PropTypes.string,
+    meta: PropTypes.shape({
+      error: PropTypes.node,
+    }),
+    input: PropTypes.shape({
+      onChange: PropTypes.func,
+      onFilter: PropTypes.func,
+      value: PropTypes.string,
+    }),
+  };
+
+  // istanbul ignore next
+  render() {
+    const { addOrganization, input: { onChange, value }, meta: { error }, index } = this.props;
+
+    return (
+      <FormattedMessage id="stripes-erm-components.organizations.clickSearchButton">
+        {placeholder => (
+          <TextField
+            error={error}
+            endControl={(
+              <Pluggable
+                aria-haspopup="true"
+                dataKey="organization"
```

Comment:
If you reuse this `dataKey`, does the plugin reset its state when opened multiple times? Ie, when you open modal, set some filters and search query, pick an org, add another org, and open the modal again, are the filters/query default or are they the ones you set previously.

Reply:
Yes, it is resetting the defaults.